web/client/epics/permalink.js

```diff
@@ -122,30 +163,20 @@ export const savePermalinkEpic = (action$, { getState = () => {} }) =>
                                     ]
                                 })
                         )
-                    ).switchMap(() =>
-                        Observable.of(
-                            updatePermalinkSettings({name}),
-                            show({
-                                id: "PERMALINK_SAVE_SUCCESS",
-                                title: "notification.success",
-                                message: "share.permalink.success"
-                            })
-                        )
-                    )
+                    ).switchMap(() => Observable.of(
+                        updatePermalinkSettings({name}),
+                        show({
+                            id: "PERMALINK_SAVE_SUCCESS",
+                            title: "notification.success",
+                            message: `permalink.${categoryCreated ? 'createAndSaveSuccess' : 'success'}`
+                        })
+                    ).do(categoryCreated = false))
```

Comment:
This do not work as you expected. In particular this always sets categoryCreated to false when thecontaining function is invoked, while it should do it after all. 
to do what you want you may have to do something like `.do(() => categoryCreated = false)`.

I suggest to avoid `categoryCreated` flag handling and emit 2 notifications, one inside the error handler that notifies that the permalink category was missing and it has been created, the other is this one, that always notifies the success of the permalink creation. 
This isolates more the error handling from the rest of the workflow, helping future maintainance.